src/accessories/platformAccessory.ts

```diff
@@ -13,7 +14,98 @@ export abstract class MiHomePlatformAccessory {
     protected readonly platform: MiHomeGatewayPlatform,
     protected readonly accessory: PlatformAccessory,
   ) {
-    // TODO
+
+    // set accessory information
+    this.accessory.getService(this.platform.Service.AccessoryInformation)!
+      .setCharacteristic(this.platform.Characteristic.Manufacturer, MANUFACTURER)
+      .setCharacteristic(this.platform.Characteristic.Model, this.accessory.context.device_type)
```

Comment:
`this.accessory.context.device.device_type`?

Reply:
Since a specific device "model" isn't exposed by the API, the `device_type` seems to be the closest thing.